owncloudApp/src/main/java/com/owncloud/android/ui/adapter/ReceiveExternalFilesAdapter.java

```diff
@@ -89,6 +86,10 @@ public Object getItem(int position) {
         }
     }
 
+    public void setNewItemVector(Vector<OCFile> newItemVector) {
+        mFiles = newItemVector;
+    }
```

Comment:
I would suggest moving `notifyDataSetChanged()` here, and update `mImmutableFilesList` too